source/linux/Demo_User_Guides/Webserver_Demo_User_Guide.rst

```diff
@@ -0,0 +1,162 @@
+.. _Webserver-Demo-User-Guide-label:
+
+############################
+Web server demo - User Guide
+############################
+
+*********
+Overview
+*********
+
+This document describes the Out of Box Web Server Demo Application, which the Linux
+SDK delivers with the |__SDK_FULL_NAME__| |__PART_FAMILY_NAME__|, applicable to boards
+such as the `BEAGL-BONE-GRN-ECO <https://www.ti.com/tool/BEAGL-BONE-GRN-ECO>`__ and
+`TMDXEVM3358 <https://www.ti.com/tool/TMDXEVM3358>`__. The main purpose of this demo is to
+showcase a glimpse of the ARM analytics AI stack integrated into the filesystem,
+providing an out-of-box experience even users do not connect a display to the board.
+
+The demo showcases a web server running on an AM335x based board, providing a web interface with three key sections:
+
+*   **Audio Classification Demo**: Users can connect a USB audio capture device to run real-time audio classification, displaying classification statistics.
+*   **Live CPU Performance Metrics**: Provides a real-time CPU usage indicator, historical usage data for the last 5 minutes, and detailed CPU information.
+*   **Documentation Links**: Offers convenient access to relevant documentation.
+
+This document provides all necessary equipment requirements and instructions.
+
+**********************
+Hardware Prerequisites
+**********************
+
+-  TI AM335x based board (e.g., `BEAGL-BONE-GRN-ECO <https://www.ti.com/tool/BEAGL-BONE-GRN-ECO>`__ and `TMDXEVM3358 <https://www.ti.com/tool/TMDXEVM3358>`__)
+   (Note: For the Beaglebone Green Eco, a display is available via an `HDMI cape <https://wiki.seeedstudio.com/BeagleBone_Green_HDMI_Cape>`__)
+-  PC (Windows or Linux)
+-  Ethernet cables
+-  Ethernet switch or ethernet router with DHCP service
+-  SD card (minimum 16GB)
+-  Audio Capture Device
+
+***********
+Get Started
+***********
+
+1.  Flash an SD card with the :file:`tisdk-default-image`. User can download the :file:`tisdk-default-image` wic
```

Comment:
We should use auto numbering wherever we can: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#lists-and-quote-like-blocks